lib/notification-issue.coffee

```diff
@@ -190,6 +191,9 @@ class NotificationIssue
           return packName unless /^\.\./.test(relativePath)
       @getPackageNameFromFilePath(filePath)
 
+    packageName = /Failed to (load|activate) (the|a) (.*) package/.exec(message)?[3]
```

Comment:
Yeah, I wonder if putting it in this payload would be sufficient, https://github.com/atom/atom/blob/666bb344647d1f96df93c875e404e32bd6c1d419/src/package.coffee#L637